src/utils/order_validation_utils.ts

```diff
@@ -0,0 +1,89 @@
+import {ExchangeContractErrs, SignedOrder} from '../types';
+import {TokenWrapper} from '../contract_wrappers/token_wrapper';
+
+export class OrderValidationUtils {
+    public static async validateFillOrderBalancesAndAllowancesAndThrowIfInvalidAsync(
```

Comment:
Let's remove `And` from the name, it makes it longer without adding any new information.